src/data/automation.ts

```diff
@@ -264,6 +264,28 @@ export type ConditionWithShorthand =
   | ShorthandOrCondition
   | ShorthandNotCondition;
 
+export const expandConditionWithShorthand = (
+  cond: ConditionWithShorthand
+): Condition => {
+  if ("condition" in cond && typeof cond.condition === "object") {
```

Comment:
Should this check for Array?
```suggestion
  if ("condition" in cond && Array.isArray(cond.condition)) {
```

Reply:
Actually I don't see this case being represented in the types.